test/Removal.t.sol

```diff
@@ -456,12 +504,18 @@ contract Removal__beforeTokenTransfer is NonUpgradableRemoval {
     );
   }
 
-  function test_zeroValueTransfer_reverts_RemovalAmountZero() external {
+  function test_zeroValueTransferToMarket_reverts_RemovalAmountZero() external {
+    address mockMarketAddress = vm.addr(1);
+    vm.store(
+      address(this),
+      bytes32(uint256(401)),
+      bytes32(uint256(uint160(address(mockMarketAddress))))
+    );
```

Comment:
can you move everything except the `super.beforeTokenTransfer` to a setup function? gives more realistic gas estimates when running `forge snapshot` as it'll exclude these things from the gas calculation (also make sure to commit the resulting `.gas-snapshot`

Reply:
moot since this function no longer depends on having a specific market address to revert on 0 amount (per our discussion to restore that behavior)